### PR TITLE
Checks/424 creating topic UI check

### PR DESCRIPTION
**Pull request by bvolodarskiy**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing applications:)

**What changes did you make?** (Give an overview)

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "X" next to an item)

- [x] No need to
- [ ] Manually(please, describe, when necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "X" next to an item, otherwise PR will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings(e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

### CODE REVIEW

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/pages/MainPage.java [hunk 1]
```diff
@@ -2,15 +2,58 @@
...
+
+  private void refreshUntil(By by){
+    int i =0;
+    do
+    {
+      refresh();
+      i++;
+      sleep(2000);
+    } while(getElements(by).size()<1 && i!=20);
+    $(by).shouldBe(Condition.visible);
```
Comment (mikementor) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/pages/MainPage.java:30]: Doesn't `$(by).waitUntil` works here instead?

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/topics/TopicTests.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    @AfterEach
+    @SneakyThrows
+    void  afterEach(){
+        steps.kafka.deleteTopic(KafkaSteps.Cluster.SECOND_LOCAL,NEW_TOPIC);
+    }
+
+    @SneakyThrows
+    @DisplayName("should create a topic")
+    @Test
+    void shouldWhen(){
```
Comment (mikementor) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/topics/TopicTests.java:25]: my bad with naming

**Review by mikementor — COMMENTED** [review]
minor comments

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/pages/MainPage.java (hunk 1 above)
Comment (bvolodarskiy) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/pages/MainPage.java:30]: for what purpose? We already waited necessary time in a loop above

Comment (mikementor) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/pages/MainPage.java:30]: instead of the loop

**Pull request merged**